.gitignore

```diff
@@ -1,4 +1,4 @@
 
 node_modules
 build
-
+package-lock.json
```

Comment:
we should not be ignoring this- 
According to npm cli docs, it needs to be committed to the repo so all install can generate identical trees
See: https://github.com/npm/cli/blob/release-6.14.7/docs/content/configuring-npm/package-lock-json.md